tests/src/test/java/alluxio/client/cli/fs/command/CpCommandIntegrationTest.java

```diff
@@ -138,6 +161,86 @@ public void copyWildcard() throws Exception {
         equals(new AlluxioURI("/copy/foobar3"), new AlluxioURI(testDir + "/bar/foobar3")));
   }
 
+  /**
+   * Tests copying a file with attributes preserved.
+   */
+  @Test
+  public void copyFileWithPreservedAttributes() throws Exception {
```

Comment:
can we also test that other attributes like TTL are *not* preserved

Reply:
Updated.